src/com/haskforce/parsing/_HaskellParsingLexer.java~

```diff
@@ -0,0 +1,1704 @@
+/* The following code was generated by JFlex 1.4.3 on 1/18/15 11:43 AM */
```

Comment:
Indeed, it was. I added a pattern to the .gitignore file to not run into this anymore. Deleted the file as well.